File: app/src/main/java/com/vikingsen/cheesedemo/ux/cheeselist/CheeseListComponent.java
package com.vikingsen.cheesedemo.ux.cheeselist;


import com.vikingsen.cheesedemo.inject.ActivityScope;

import dagger.Subcomponent;

@ActivityScope
@Subcomponent(modules = CheeseListModule.class)
public interface CheeseListComponent {
    void inject(CheeseListActivity target);
}

File: app/src/main/kotlin/com/vikingsen/cheesedemo/ux/cheeselist/CheeseListComponent.kt
package com.vikingsen.cheesedemo.ux.cheeselist


import com.vikingsen.cheesedemo.inject.ActivityScope

import dagger.Subcomponent

@ActivityScope
@Subcomponent(modules = arrayOf(CheeseListModule::class))
interface CheeseListComponent {
    fun inject(target: CheeseListActivity)
}
